packages/flutter/lib/src/material/expansion_panel.dart

```diff
@@ -84,6 +86,34 @@ class ExpansionPanel {
   ///
   /// Defaults to false.
   final bool isExpanded;
+
+}
+
+///An extension of the expansion panel class that allows for radio functionality.
+///
+///This type of panel will automatically handle closing and opening itself, so
+///setting the [isExpanded] value on these panels will have no effect. A unique
+///identifier [value] must be assigned to each panel.
+class ExpansionPanelRadio extends ExpansionPanel {
+
+  ///A subclass of expansion panel that allows for radio functionality.
+  ///
+  ///A key must be passed into the constructor. The [headerBuilder], [body],
+  /// and [isExpanded] arguments must not be null.
```

Comment:
This mentions a key, but there is not key param (value?) and mentions isExpanded but that doesn't exist.